guides/contributor/genai.md

```diff
@@ -0,0 +1,83 @@
+# Generative AI Contribution Policy
```

Comment:
Can we link to this document from the [guides/README.md](https://github.com/open-telemetry/community/tree/main/guides) to make it more discoverable?

Reply:
Done